RecipesBase/src/RecipesBase.jl

```diff
@@ -123,7 +123,7 @@ function create_kw_body(func_signature::Expr)
             push!(
                 cleanup_body.args,
                 :(
-                    RecipesBase.is_key_supported($(QuoteNode(k))) ||
+                    $is_key_supported($(QuoteNode(k))) ||
```

Comment:
Because there is no need to interpolate the module at method _calls_. interpolating the function directly is more straightforward and clear than interpolating the module and then looking up the function within the module.

Reply:
This is inconsistent with https://github.com/LilithHafner/Plots.jl/blob/patch-1/RecipesBase/src/RecipesBase.jl#L90 then.